### DISCUSSION TITLE
baseline file for windows, linux and macos

### DISCUSSION THREAD

**Discussion by DetachHead** [KotlinIsland/basedmypy#132] [Ideas]

**Comment by KotlinIsland** [comment]
What if all the baselines were aggregated into a single file?

**Reply by KotlinIsland** [reply]
Related #86

**Comment by Zeckie** [comment]
In what cases does mypy give different errors for different operating systems?

**Reply by KotlinIsland** [reply]
```py
if sys.platform == "win32":
    a: int = ""
```
Mypy won't show any errors when you run against this code.
